src/integration-tests/bash/run.sh

```diff
@@ -2502,9 +2500,31 @@ function wait_for_operator_shutdown {
     fi
   done
   if [ ${deleted} == false ]; then
-    fail 'operator fail to be deleted'
+    fail 'the operator helm release ${release} failed to be deleted'
   else
-    trace "operator namespace ${name} has been deleted"
+    trace "the operator helm release ${release} has been deleted"
+  fi
+}
+
+function wait_for_operator_deployment_deleted {
+  namespace=$1
+  deployment="weblogic-operator"
+  deleted=false
+  iter=1
+  trace "waiting for the operator deployment ${deployment} in the namespace ${namespace} to longer exist"
+  while [ ${deleted} == false -a $iter -lt 101 ]; do
+    helm get deployment -namespace ${namespace} ${deployment}
```

Comment:
i am not familiar with helm get deployment, but could you double check the syntax? I usually see namespace option with 2 dashes instead of 1. Thanks

Reply:
Good catch - I meant 'kubectl delete ...'